src/main.cpp

```diff
@@ -67,6 +67,8 @@ static void lPrintVersion() {
 #endif
     printf("    [--cpu=<type>]\t\t\tAn alias for [--device=<type>] switch\n");
     printf("    [-D<foo>]\t\t\t\t#define given value when running preprocessor\n");
+    printf("    [--darwin-version-min=<major.minor>]\tSet the minimum macOS/iOS version required for the "
```

Comment:
I'm wondering if we should put it under `ISPC_MACOS_TARGET || ISPC_IOS_TARGET`. Any thoughts?

Reply:
Added